nix/modules/tezos-node.nix

```diff
@@ -1,14 +1,13 @@
 # SPDX-FileCopyrightText: 2021 Oxhead Alpha
 # SPDX-License-Identifier: LicenseRef-MIT-OA
 
-{config, lib, pkgs, ...}:
+{ config, lib, pkgs, ... }:
```

Comment:
I think you can add `inputs` here and then use `inputs.serokell-nix` below for the `jsonConfig` type

Reply:
I don't think it's possible. But it could be easily added to pkgs, I just did it the wrong way